src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -529,16 +530,17 @@ private bool SyncOptionalWorkloads()
 
             if (!isCustomHive)
             {
-                string sdkVersion = EnvironmentSettings.Host.Version;
+                string sdkVersion = EnvironmentSettings.Host.Version.Substring(1); // Host.Version (from SDK) has a leading "v" that need to remove.
 
                 try
                 {
                     List<InstallationRequest> owInstallationRequests = new List<InstallationRequest>();
                     Dictionary<string, string> owInstalledPkgs = new Dictionary<string, string>();  // packageId -> packageVersion
                     HashSet<string> owSyncRequestsPackageIds = new HashSet<string>();
                     TemplateLocator optionalWorkloadLocator = new TemplateLocator();
+                    string dotnetPath = Path.GetDirectoryName(Path.GetDirectoryName(_paths.Global.BaseDir));
```

Comment:
minor:  consider ```DirectoryInfo::Name```

Reply:
"This Name property returns only the name of the directory, such as "Bin". To get the full path, such as "c:\public\Bin", use the FullName property." That is not what we want

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.io.directoryinfo.name?view=netcore-3.1

We want something like `c:/program files/dotnet`